src/test/java/com/box/sdk/BoxFolderTest.java

```diff
@@ -1360,6 +1360,34 @@ public void testUploadFileWithDescriptionSucceeds() throws IOException {
         Assert.assertEquals(fileDescription, file.getDescription());
     }
 
+    @Test
+    @Category(UnitTest.class)
+    public void testGetFolderItemsWithSort() throws IOException {
+        String result = "";
+        final String folderID = "12345";
+        final String sortField = "name";
+        final String folderItemsURL = "/folders/" + folderID + "/items/";
+
+        result = TestConfig.getFixture("BoxFolder/GetFolderItemsWithSort200");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.get(WireMock.urlPathEqualTo(folderItemsURL))
+                .withQueryParam("sort", WireMock.equalTo("name"))
+                .withQueryParam("direction", WireMock.equalTo(BoxFolder.SortDirection.ASC.toString()))
```

Comment:
In tests, I'd prefer to use direct values as much as possible (`"ASC"` here).  `BoxFolder.SortDirection.ASC.toString()` will never fail, because it's just repeating exactly what the method does, and it doesn't give us much confidence that we actually know what's being sent over the wire to the API.